addon/utils/date.js

```diff
@@ -0,0 +1,10 @@
+export function formatDate(date) {
+  if (!(date instanceof Date) || isNaN(date)) {
+    return '';
+  }
+
+  let day = `${date.getDate()}`.padStart(2, '0');
```

Comment:
Great find on the padStart function! I didnt know that existed, would have been handy on other components like the timepicker that I created in appuniversum :p